CONTRIBUTING.md

```diff
@@ -2,17 +2,19 @@
 
 ## The basic
 
+A lot of discussions about ideas take place in the [Issues](https://github.com/datasciencebr/serenata-de-amor/issues) section. There you can catch up with what's going on and also suggest new ideas.
+
 1. _Fork_ this repository
-2. Create your feature branch: `$ git checkout -b my-new-feature`
-3. Commit your changes: `$ git commit -am 'Add some feature'`
-4. Push to the branch to your fork: `$ git push origin my-new-feature`
-5. Create a new _Pull Request_
+2. Create your branch: `$ git checkout -b xx-new-stuff` where `xx` are your initials (e.g `jd` for John Doe)
```

Comment:
When forking, the branch turns unnecessary since it's already owned by the contributor and no other user.